include/deal.II/hp/fe_collection.h

```diff
@@ -188,6 +188,27 @@ namespace hp
     bool hp_constraints_are_implemented () const;
 
     /**
+     * Try to find a least dominant finite element inside this FECollection
+     * which dominates other finite elements provided as fe_indices in @p fes .
+     * For example, if FECollection consists of {Q1,Q2,Q3,Q4} and we are looking
+     * for the least dominant FE for Q3 and Q4 (@p fes is {2,3}), then the
+     * answer is Q3 and therefore this function will return its index in
+     * FECollection, namely 2.
+     *
+     * If we were not able to find a finite element, the function returns
+     * numbers::invalid_unsigned_int .
+     *
+     * Note that for the cases like when FECollection consists of
+     * {FE_Nothing(),Q1} with @p fes = {1}, the function will not find the most
+     * dominating element as the default behavior of FE_Nothing is to return
+     * FiniteElementDomination::no_requirements when comparing for face domination.
+     * This, therefore, can't be considered as a dominating element in the sense
+     * defined in FiniteElementDomination .
```

Comment:
i don't think it's about 1 element in the set, it's about FE_Nothing by default returning `no_reqirements`. That's the reason why it can't be considered as dominating, per se. 

After another PR is merged (https://github.com/dealii/dealii/pull/1484), that would be an option. 

Perhaps i was not clear in the documentation...